### PR TITLE
Test module

### PR DESCRIPTION
**Pull request by benrudolph**
Just opening this so I can run the tests on travis. been having trouble getting jython 2.7 to run on travis. also there's a bug in jython 2.5.3 that makes this not possible with that version. basically it can't find any of the test suites because `issubclass` fails to return correctly when testing if the suite is a subclass of TestCase
elf: @snopoke

### CODE REVIEW

File: touchforms/backend/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
+
+from .test_server import XFormServerTest
```
Comment (dimagimon) [inline touchforms/backend/tests/__init__.py]: F401 'XFormServerTest' imported but unused

File: touchforms/backend/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+from .test_xformplayer import XFormPlayerTest
```
Comment (dimagimon) [inline touchforms/backend/tests/__init__.py]: F401 'XFormPlayerTest' imported but unused

**Comment by snopoke** [comment]
Seriously loathing jython right now

**Comment by benrudolph** [comment]
tell me about it. killing me slowly. this probably should have been built in java

**Comment by snopoke** [comment]
Still could be.....

**Pull request closed without merging**